### ISSUE TITLE
Imports are duplicated when using barrels 

### ISSUE THREAD

**Issue by NateMay** [buehler/typescript-hero#208]
#### Description

When I create import barrels in an `index.ts` file, all of the imports in that index file are imported duplicately. 

##### Steps to reproduce

1. create an import [barrel](http://tattoocoder.com/angular2-barrels/)
2. add an exported member of the barrel to another file
3. run the typescript hero command to add missing imports to the open document

**Comment by buehler** [comment]
I think this one could be related to #175

**Comment by NateMay** [comment]
I think #175 is a name collision issue. I've had to rename exports and filenames before to prevent such collisions. 

instead this happens not because there are multiple constructs with the same name, but because the same construct is exported from multiple files via a barrel

index.ts (my barrel at the services level)
```
export * from './auth/auth.service';
export * from './hero/hero.service';
```

In this example, I barrel my services so that I can import them like such:

import * as svc from '../../services'

but TypeScript Hero also resolves like this:

```
import * as svc from '../../services'
import { HeroService, AuthService } from '../../services/
import { HeroService } from '../../services/hero/hero.service'
import { AuthService } from '../../services/auth/auth.service'
```

**Comment by swissmanu** [comment]
this is so annoying. would love to fix it if i knew how 🤔😅

**Comment by buehler** [comment]
well.. I also found out, this only happens on windows :-/

**Comment by buehler** [comment]
Has something to do with the stupid `/` and `\`

**Comment by buehler** [comment]
I'm having my hopes up for https://github.com/buehler/node-typescript-parser/pull/37

**Comment by buehler** [comment]
Closed by #329
